index.js

```diff
@@ -1,47 +1,163 @@
 'use strict';
-const util = require('util');
-const fs = require('fs');
-const chalk = require('chalk');
+
 const Jimp = require('@sindresorhus/jimp');
-const termImg = require('term-img');
+const chalk = require('chalk');
+const {string: termImg} = require('term-img');
+const terminalCharWidth = require('terminal-char-width');
```

Comment:
I don't trust this module. Look at its source. And I don't understand why it's used in the first place.

Reply:
I've seen the source code and didn't find anything strange... Can you explain your concerns? I'm using it to get the width/height relationship of the terminal characters (0.725 on Windows, 0.5 on everything else, like GNU Unifont) so the aspect ration can be preserverd.